frontend/src/metabase/core/components/Alert/Alert.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { ReactNode } from "react";
 import { AlertIcon, AlertRoot } from "./Alert.styled";
 
-export type AlertVariant = "info" | "error";
+export type AlertVariant = "info" | "warning" | "error";
```

Comment:
maybe while we're touching this component, maybe it's a good time to give it a few simple unit tests?